Bin/Documentation.php

```diff
@@ -309,7 +316,40 @@ public function main()
             }
         }
 
-        echo "\n", 'Open file://', $workspace, '/index.html', '.', "\n";
+        $pathname = escapeshellarg('file://' . $workspace . '/index.html');
+
+        echo "\n";
+
+        if (true === $open) {
+            if (isset($_SERVER['BROWSER'])) {
```

Comment:
You have your answer here: https://unix.stackexchange.com/questions/213367/where-do-editor-pager-browser-environment-variables-come-from.